packages/flutter/test/services/text_formatter_test.dart

```diff
@@ -7,6 +7,13 @@ import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  test('test const constructor', () {
+    const testValue1 = TextInputFormatter();
```

Comment:
TextInputFormatter is an abstract class and cannot be instantiated.

Reply:
@goderbauer 
Thanks for your comment. Should I write tests for const constructors of subclasses instead?